packages/shared/src/components/filters/MyFeedHeading.tsx

```diff
@@ -64,25 +47,26 @@ function MyFeedHeading({
       >
         {!isMobile ? feedFiltersLabel : null}
       </FeedSettingsButton>
-      {isExtension &&
-        checkHasCompleted(ActionType.FirstShortcutsSession) &&
-        !showTopSites &&
-        isShortcutsUIV1 && (
-          <Button
-            size={ButtonSize.Medium}
-            variant={isLaptop ? ButtonVariant.Float : ButtonVariant.Tertiary}
-            className="mr-auto"
-            onClick={() => {
-              toggleShowTopSites();
-            }}
-            icon={<PlusIcon />}
-            iconPosition={
-              shouldUseListFeedLayout ? ButtonIconPosition.Right : undefined
+      {showToggleShortcuts && (
+        <Button
+          size={ButtonSize.Medium}
+          variant={isLaptop ? ButtonVariant.Float : ButtonVariant.Tertiary}
+          className="mr-auto"
+          onClick={() => {
+            if (isOldUserWithNoShortcuts) {
+              completeAction(ActionType.FirstShortcutsSession);
+              return;
```

Comment:
After completing the action, it seems that we won't be doing anything here. Is that intended?

Reply:
Yes, completing action turn `isOldUserWithNoShortcuts` into a standard user and enable the shortcuts ✔️